src/index.js

```diff
@@ -1 +1,32 @@
-export default () => true
+import defaultsDeep from 'lodash.defaultsdeep';
+import pickBy from 'lodash.pickby';
+
+class Stipulate {
+
+  constructor(options) {
+    this.baseOptions = options;
+  }
+
+  prefix(request) {
+    return request;
+  }
+
+  postfix(response) {
```

Comment:
I'm still not following; what do you mean by existing outside of fetch? My train of thought is:
- stipulate makes requests, and receives responses(doesn't matter whether you're talking about the classes specific to fetch or the concepts in general, same holds true)
- the "before" function is called before sending off a request(again, both are relevant) and gets passed the Request instance(fetch class)
- the "after" function is called after receiving a response(both relevant) and gets passed the Response instance(fetch class)

the upside i see to `onResponse` is that likewise it can be interpreted as either "on response" or "on Response" and both are accurate. 

perhaps `afterResponse` makes even more sense, as we have already received a response, and are performing an action after that fact. that would also give us nice symmetry if we used `beforeRequest` as well. 

Thoughts?